docs/rules/ordered-list-marker-sequence.md

```diff
@@ -106,7 +113,79 @@ paragraph
 
 ## 🔧 Options
 
-This rule has no options.
+This rule has an optional configuration object with the following property:
+
+- `increment` (default: `"always"`): Controls how list markers should be numbered.
+  - `"always"`: Requires sequential numbering (1, 2, 3, ...)
+  - `"never"`: Requires all items to use `1.` or `1)` as the marker
```

Comment:
0 must also be allowed.
The rule that enforces consistency between 0 and 1 is `ordered-list-marker-start`.